ui/js/controllers/logviewer.js

```diff
@@ -217,10 +217,8 @@ logViewerApp.controller('LogviewerCtrl', [
 
             $scope.logProperties = [];
             ThJobModel.get($scope.repoName, $scope.job_id).then(function(job) {
-                var jobStr = thJobSearchStr(job);
-
                 // set the title of the browser window/tab
-                $scope.logViewerTitle = "Log for " + jobStr;
+                $scope.logViewerTitle = job.get_title();
```

Comment:
This is a minor UI change that I snuck in. I think the "Log for..." part is redundant, especially since we now have a proper log icon in the tab. If you feel like it should be part of a separate commit I can take it out.